packages/gatsby-theme-docs/src/layouts/internals/layout-page-release-notes-filters.js

```diff
@@ -32,7 +32,6 @@ const SlidingContainer = styled.div`
 const GridContainer = styled.div`
   display: none;
   border-left: 1px solid ${designSystem.colors.light.borderPrimary};
-  margin-top: ${designSystem.dimensions.spacings.wide};
```

Comment:
Is there a reason why we have this huge margin over the release note filter?

Reply:
The spacing between page titel (in that case "Platform Release Notes") and the beginnning of the actual page content is supposed to be mirrored in the left column. 

In the new concept with the search bar,  the placeholder "line" in the grid for the search bar could become that space.  It's smaller than currently but I think the spacing of the release notes should be identical to the content pages.  

I suggest to you try to get the release notes into a spacing and grid setup as similar to the content pages as possibe and then show it to sven (need to do a video call because it's not on staging)

Reply:
Yes, I had a meeting with sven yesterday. He likes the current implementation. I also showed hin the changes for the release note pages. Thanks for the explanation 👍